modules/gapi/src/streaming/onevpl/cfg_params_parser.cpp

```diff
@@ -44,6 +41,18 @@ struct ParamCreator<mfxVariant> {
         ret.Data.U32 = value;
         return ret;
     }
+    mfxVariant create_impl(const std::string&, mfxI64 value) {
+        mfxVariant ret;
+        ret.Type = MFX_VARIANT_TYPE_I64;
+        ret.Data.I64 = value;
+        return ret;
+    }
+    mfxVariant create_impl(const std::string&, mfxU64 value) {
```

Comment:
Why are create_impl functions not a template? Is only three mfx-types expected? Does template bring only difficulties?